Dockerfile

```diff
@@ -216,7 +217,7 @@ ARG LIBGSM_COMMIT=98f1708fb5e06a0dfebd58a3b40d610823db9715
 RUN \
   git clone "$LIBGSM_URL" && \
   cd libgsm && git checkout $LIBGSM_COMMIT && \
-  # Makefile is garbage, hence use specific compile arguments and flags
+  # Makefile is hard to use, hence use specific compile arguments and flags
```

Comment:
Hahah, that must have been my wording there with the garbage! Good on you to clean up!

Reply:
hehe didn't check the source! but it sounded a bit harsh

Reply:
yes, was from #155 i recognize my language hahha. Take care brother